luatest/server.lua

```diff
@@ -858,12 +858,13 @@ function Server:wait_for_vclock(vclock)
     end
 end
 
---- Wait until all own data is replicated and confirmed by the given server.
+--- Wait until all self data is replicated and the downstream knows,
```

Comment:
What about this description?
```
Wait for the given server to reach at least the same vclock as the local
server. Not including the local component, of course.
```

In this case, it will be consistent with `wait_for_vclock_of`.